journal-writings/modernized/04-chapter-2.adoc

```diff
@@ -77,7 +77,7 @@ and a little retailing of goods, get a living in a plain way,
 without the load of great business.
 
 I mentioned it to my employer, and we soon agreed on terms; and then,
-when I had leisure from the affairs of merchandize, I worked with this man.
+when I had leisure from the affairs of merchandise, I worked with this man.
```

Comment:
Gosh, staying consistent with the above changes I recommended is a little tough. How about:

```ASCIIDOC
when I had leisure from the affairs of the mercantile business, I worked with this man.
```